examples/torch/torch-reference.py

```diff
@@ -0,0 +1,110 @@
+"""Pytorch reference turorial script from (CIFAR10 classifier):
+https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html#sphx-glr-beginner-blitz-cifar10-tutorial-py
```

Comment:
We could just include the link to the tutorial in the README without including the whole scripts:
- less code to maintain
- data scientists interested in integration torch with substra will most probably already have knowledge of pytorch.

Reply:
I think we should keep the original file here.

The online tutorial may change in the future and we want to keep a version of it.
It will also allow to easily compare and validate the two versions (w and wo Substra).

IMO it's easier to compare two python scripts than comparing an online example and a script.